.github/workflows/tests-and-coverage.yml

```diff
@@ -56,8 +51,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: build postgres
+        run: |
+          docker build ./database -t pgvalor
       - name: setup back end test env
-        run: docker compose -p valor -f docker-compose.yml --env-file ./api/.env.testing up --build -d
+        run: docker compose -p valor -f docker-compose.yml -f docker-compose.cicd-override.yml --env-file ./api/.env.testing up --build -d
```

Comment:
you're building the container above with tag pgvalor but hten this ci/cd override is going to pull it from github container registry so whats the point of the build? or do you want docker compose to get it from the newly built one? in the latter you need to give it the right tag

Reply:
The opposite should occur. The override will tell it to look locally for the container.

Reply:
the override has `docker.io/library/pgvalor`. will that even exist?? we're not pushing to docker.io right?

Reply:
Ill remove the prefix. I found that the local repo is reflected by this, i guess it would be a problem if we dont build it first